graph_def_editor/visualization/jupyter_helper.py

```diff
@@ -0,0 +1,69 @@
+"""Helper methods to display gde graph in Jupyter Notebook or Colab."""
```

Comment:
Can you please add a copyright header to the top of this file? You can copy it from any other file in the project.